src/test/java/com/exasol/dbbuilder/dialects/exasol/VirtualSchemaTest.java

```diff
@@ -86,6 +86,15 @@ void testGetConnectionDefinition(@Mock final ConnectionDefinition connectionDefi
                 sameInstance(connectionDefinitionMock));
     }
 
+    @Test
+    void testDebugProperties() {
+        System.setProperty(VirtualSchema.DEBUG_ADDRESS, "1.2.3.4:3000");
+        System.setProperty(VirtualSchema.DEBUG_LOG_LEVEL, "WARN");
+        final Map<String, String> properties = this.builder.build().getProperties();
+        assertThat(properties.get("DEBUG_ADDRESS"), equalTo("1.2.3.4:3000"));
+        assertThat(properties.get("LOG_LEVEL"), equalTo("WARN"));
```

Comment:
Use JUnit Pioneer to automatically clear the system properties after the test: https://junit-pioneer.org/docs/system-properties/

Reply:
Wow! Didn't know that - cool!
Done.